server/src/main/java/org/apache/iotdb/db/qp/strategy/optimizer/ConcatPathOptimizer.java

```diff
@@ -180,34 +192,26 @@ private void concatSelect(List<PartialPath> fromPaths, SelectOperator selectOper
       }
     }
 
-    removeStarsInPath(allPaths, afterConcatAggregations, selectOperator);
+    removeStarsInPath(allPaths, afterConcatAggregations, selectOperator, limit, offset);
   }
 
   /**
    * Make 'SLIMIT&SOFFSET' take effect by trimming the suffixList and aggregations of the
    * selectOperator.
    *
-   * @param seriesLimit  is ensured to be positive integer
    * @param seriesOffset is ensured to be non-negative integer
    */
-  private void slimitTrim(SelectOperator select, int seriesLimit, int seriesOffset)
+  private void slimitTrim(SelectOperator select, int seriesOffset)
       throws LogicalOptimizeException {
```

Comment:
From my perspective, limit is done in concatselect() while this method is only for trimming the `offset` of aggregations. 
If I'm right, please modify the method name and java doc of it.

Reply:
fix